src/Sylius/Bundle/AdminBundle/Resources/config/grids/payment.yml

```diff
@@ -6,9 +6,12 @@ sylius_grid:
                 options:
                     class: "%sylius.model.payment.class%"
             fields:
-                id:
-                    type: string
-                    label: sylius.ui.id
+                createdAt:
```

Comment:
What do you think about placing `createdAt` as the last element in the table?

Reply:
I was the one to suggest placing it as the first one. Consistent with the only other entity that has date but does not have its separate number - product reviews : https://master.demo.sylius.com/admin/product-reviews/

Reply:
Whatever suits you best. I just thought that once payments are sorted by day, related order number or channel is more valuable information for me.